rkt/image/fetcher.go

```diff
@@ -205,12 +205,12 @@ func (f *Fetcher) getRemoteForURL(u *url.URL) (*imagestore.Remote, error) {
 		return nil, nil
 	}
 	urlStr := u.String()
-	if rem, ok, err := f.S.GetRemote(urlStr); err != nil {
+	rem, err := f.S.GetRemote(urlStr)
+	if err != nil && err != imagestore.ErrRemoteNotFound {
```

Comment:
@sgotti this is because the tests were structured using the unrefactored code. After refactoring some tests might look a bit awkward and the old way might look better